clientlibs/java/src/main/java/org/upgradeplatform/client/ExperimentClient.java

```diff
@@ -274,8 +274,13 @@ public void markExperimentPoint(final String experimentPoint, String condition,
 
 	public void markExperimentPoint(final String experimentPoint, String experimentId, String condition,
 			final ResponseCallback<MarkExperimentPoint> callbacks) {
+		markExperimentPoint(experimentPoint, "", condition, MarkedDecisionPointStatus.CONDITION_APPLIED, callbacks);
```

Comment:
same as in #507 WRT the TS client, I think users of the Java client should be required to supply a status -- don't provide a default, make the developer who's integrating with UpGrade think this through.

Reply:
@jerith666 made the appropriate change! Take another look and let us know if you're comfortable with this code going to production :)